### ISSUE TITLE
Sound of the texts in the questgiver does not work in 9.0.2.

### ISSUE THREAD

**Issue by osleyder85** [TrinityCore/TrinityCore#25770]
**Description:**
I'm testing various Island of Exile npcs and when the quest is accepted, they launch the text but no sound is heard.

```
-- Captain Garrick SAI
SET @ENTRY := 156626;
UPDATE `creature_template` SET `AIName`="SmartAI" WHERE `entry`=@ENTRY;
DELETE FROM `smart_scripts` WHERE `entryorguid`=@ENTRY AND `source_type`=0;
INSERT INTO `smart_scripts` (`entryorguid`,`source_type`,`id`,`link`,`event_type`,`event_phase_mask`,`event_chance`,`event_flags`,`event_param1`,`event_param2`,`event_param3`,`event_param4`,`action_type`,`action_param1`,`action_param2`,`action_param3`,`action_param4`,`action_param5`,`action_param6`,`target_type`,`target_param1`,`target_param2`,`target_param3`,`target_x`,`target_y`,`target_z`,`target_o`,`comment`) VALUES
(@ENTRY,0,0,0,19,0,100,0,55122,0,0,0,1,0,0,0,0,0,0,1,0,0,0,0,0,0,0,"Captain Garrick - On Quest 'Murloc Mania' Taken - Say Line 0 for duration 0 not using Talktarget - Target: Self");

DELETE FROM `creature_text` WHERE `CreatureID`=@ENTRY;
INSERT INTO `creature_text` (`CreatureID`, `GroupID`, `ID`, `Text`, `Type`, `Language`, `Probability`, `Emote`, `Duration`, `Sound`, `BroadcastTextId`, `comment`) VALUES
(@ENTRY, 0, 0, 'Murlocs are dangerous in large numbers. Take back our supplies, but avoid fighting too many of them at once.', 12, 0, 100, 0, 0, 152737, 178507, 'Captain Garrick to Player');
```
Example in the parsed.txt
```
ServerToClient: SMSG_DB_REPLY (0x2594) Length: 194 ConnIdx: 0 Time: 12/22/2020 19:13:42.109 Number: 6054
TableHash: 35137211 (BroadcastText)
RecordID: 178507
Timestamp: 12/23/2020 00:13:42
Status: 1 (Valid)
Size: 177
Text: 
Text1: Murlocs are dangerous in large numbers. Take back our supplies, but avoid fighting too many of them at once.
ID: 178507
LanguageID: 0
ConditionID: 0
EmotesID: 0
Flags: 0
ChatBubbleDurationMs: 0
[0] SoundEntriesID: 0
[1] SoundEntriesID: 152737
[0] EmoteID: 0
[1] EmoteID: 0
[2] EmoteID: 0
[0] EmoteDelay: 0
[1] EmoteDelay: 0
[2] EmoteDelay: 0

ServerToClient: SMSG_DB_REPLY (0x2594) Length: 123 ConnIdx: 0 Time: 12/22/2020 19:13:33.719 Number: 5922
TableHash: 35137211 (BroadcastText)
RecordID: 178506
Timestamp: 12/23/2020 00:13:34
Status: 1 (Valid)
Size: 106
Text: 
Text1: $n! Thank the Light! Are you injured?
ID: 178506
LanguageID: 0
ConditionID: 0
EmotesID: 0
Flags: 0
ChatBubbleDurationMs: 0
[0] SoundEntriesID: 0
[1] SoundEntriesID: 152736
[0] EmoteID: 0
[1] EmoteID: 0
[2] EmoteID: 0
[0] EmoteDelay: 0
[1] EmoteDelay: 0
[2] EmoteDelay: 0
```
**Steps to reproduce the problem:**
1. Create a new player.
2. .debug play sound 152737 (It does not work)
3. .debug play sound 152736 (It does not work)
4. .debug play sound 153036 (It does not work)
5. .debug play sound 55248  (work)
6. .debug play sound 55246  (work)

**Branch(es):** master

**TC rev. hash/commit:**  89cfced

**Operating system:**  Windows 10 Pro version 20H2, Version OS: 19042.662

**Comment by MaxtorCoder** [comment]
Really, literally got reported 2 hours ago.

**Comment by offl** [comment]
> Really, literally got reported 2 hours ago.

Compare them first.

**Comment by MaxtorCoder** [comment]
> 
> 
> > Really, literally got reported 2 hours ago.
> 
> Compare them first.

It's the exact same issue lol, just not the SAI. It also has the same issue topic.

**Comment by FCondeNR** [comment]
my issue was close, thats the ending of my issue and thats more elavorated, theres a missing sound, or encryptated, theres a change of the way to retrieve the sound on shadowlands, AND THAT AFFECT TO ALL SOUNDS ON SHADOWLANDS

**Comment by offl** [comment]
> It's the exact same issue lol, just not the SAI. It also has the same issue topic.

First is not completed after 3 hours, with wrong description, without repro steps and examples.

**Comment by FCondeNR** [comment]
The sound its on SoundentryKit, but seems like a error of extractors than isnt extracting correctly, or need a key for extract, Im pretty sure its a error of extractors dbc

![WoW tools  Database browser - Google Chrome_2](https://user-images.githubusercontent.com/67517462/103177215-bc752000-4878-11eb-91b0-0d4a8b82eca8.jpg)

**Comment by illfated** [comment]
@FCondeNR : pro tip for use of MarkDown code blocks on GitHub (GitHub Flavored MarkDown — GFM); Use syntax highlighting:
- https://guides.github.com/features/mastering-markdown/#GitHub-flavored-markdown
- https://docs.github.com/en/free-pro-team@latest/github/writing-on-github/creating-and-highlighting-code-blocks#syntax-highlighting

TL;DR: in your case, you can add **sql** right after your first code block fence ( **```** ) to highlight the SQL keywords.

**Comment by ZaDarkSide** [comment]
Recently found out that Core needs some data in `hotfixes` DB, table `hotfix_optional_data` to play the sounds correctly.
